OneSignalSDK/onesignal/src/main/java/com/onesignal/OSInAppMessage.java

```diff
@@ -137,22 +189,86 @@ boolean takeActionAsUnique() {
         return actionTaken = true;
     }
 
-    public double getDisplayDuration() {
+    double getDisplayDuration() {
         return displayDuration;
     }
 
-    public void setDisplayDuration(double displayDuration) {
+    void setDisplayDuration(double displayDuration) {
         this.displayDuration = displayDuration;
     }
 
+    int getDisplaysQuantity() {
+        return displaysQuantity;
+    }
+
+    void setDisplaysQuantity(int displaysQuantity) {
+        this.displaysQuantity = displaysQuantity;
+    }
+
+    void increaseDisplayQuantity() {
+        this.displaysQuantity++;
```

Comment:
`increaseDisplayQuantity` should be `incrementDisplayQuantity`

increase could be any quantity, increment means to add 1 to a current value